keyboards/handwired/uthol/keymaps/oled/anim_loader.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 Uthol
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//Animation loader file
+#include "helix.c" //animation file
```

Comment:
c files should not be included like this. 

they should be added with `SRC` in rules, and header files included.